demo

```diff
@@ -4,6 +4,13 @@
 CLIENT="-f docker-compose.client.yml"
 [ -n "${NO_CLIENT}" ] && unset CLIENT
 
+if [ ! -z "$SCALE" ]; then
```

Comment:
You can pass scale directly in arguments for `demo` like this: `./demo scale mender-client=3`